test/DynamoCoreTests/ListAtLevelTests.cs

```diff
@@ -73,5 +73,12 @@ public void TestUseListStructureAndLacing2()
             RunModel(testFilePath);
             AssertPreviewValue("ae89387d-77b8-49e2-b59a-26a4b0a2ae9b", new[] { new[] { "3a", "7b" }, new[] { "11c", "15c" } });
         }
+        [Test]
+        public void TestUseListAtLevelOnFunctionCall()
+        {
+            string testFilePath = Path.Combine(listTestFolder, "testfunctioncall.dyn");
+            RunModel(testFilePath);
+            AssertPreviewValue("b3cd769e-0f9d-4b4f-ac86-53a07edaba24", new[] { new[] { new[] { 5, 6 } }, new[] { new[] { 5, 6 }, new[] { 7, 8 } }, new[] { new[] { 5, 6 }, new[] { 7, 8 } }, new[] { new[] { 5, 6 }, new[] { 7, 8 } } });
+        }
```

Comment:
Can we also have a pure language level test for this here: https://github.com/DynamoDS/Dynamo/blob/master/test/Engine/ProtoTest/AtLevel/AtLevelTest.cs?

Reply:
Thanks! I didn't notice those. Will do.